### DISCUSSION TITLE
[Project beta] Search by issue number

### DISCUSSION THREAD

**Discussion by ronnyroeller** [community/community#18095] [Projects and Issues]
We internally refer to issues typically by their number. Right now, entering the issue number in the search box doesn't return the issues. This makes it quite hard to find them in the  board.

**Comment by oalsing** [comment]
👍 Same for us! Would be great to have this implemented.

**Comment by adamalton** [comment]
This is especially annoying as the page intercepts the `cmd + F` keyboard shortcut to activate the search box in the page rather than the browser's search tool. Currently, the functionality is worse than if the search feature hadn't been added at all!

**Comment by cheenamalhotra** [comment]
Please provide issue number as a column for sorting.
